lambda/src/main/resources/db/migration/V119__getConsignment_further_indexes.sql

```diff
@@ -0,0 +1,15 @@
+-- Add index in tables File FileMetadata and FileStatus to improve getConsignment query 
+
+CREATE INDEX IF NOT EXISTS file_allfields_idx 
+ON public."File" 
+USING btree ("ConsignmentId", "FileId", "FileType","FileName","ParentId","UserId","Datetime","ChecksumMatches");
```

Comment:
Could you make sure there is a space between each column name, just for consistency